docs/adding_packages/conandata_yml_format.md

```diff
@@ -172,7 +172,7 @@ An example of a full patch description could be: `port to Android: update config
 
 #### patch_type
 
-_Required_
+_Recommended_
```

Comment:
mental note that the linter needs to stop saying this is required and just show this a recommendation